src/doom/p_sight.c

```diff
@@ -16,9 +16,10 @@
 //	LineOfSight/Visibility checks, uses REJECT Lookup Table.
 //
 
-
+#include <stdlib.h> // abs()
```

Comment:
Please remove comment.

Reply:
Turns out the #include isn't needed at all any more, so removed in https://github.com/chocolate-doom/chocolate-doom/pull/1205/commits/77f49e5ea5aae45fe904306a9364018c0fc23b71 .